ydb/tests/functional/tpc/canondata/test_init.TestClickbenchInit.test_s1_s3/s1_s3

```diff
@@ -0,0 +1,127 @@
+Init tables ...
+--!syntax_v1
+
+            CREATE EXTERNAL DATA SOURCE `/Root/db/Root/db/clickbench/s1_s3_external_source` WITH (
```

Comment:
Why so many spaces at the beginning? Probably shifted to the right somewhere in code